jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaHandlerResolver.kt

```diff
@@ -107,6 +112,25 @@ class JavaLambdaHandlerResolver : LambdaHandlerResolver {
             null
         }
 
+    private fun findHandlersByClass(clz: PsiClass): Set<String> {
+        if (!clz.canBeInstantiatedByLambda()) {
+            return emptySet()
+        }
+
+        val handlers = mutableSetOf<String>()
+        if (clz.implementsLambdaHandlerInterface()) {
+            clz.qualifiedName?.let { handlers.add(it) }
+        }
+
+        handlers.addAll(clz.allMethods
```

Comment:
Maybe I've mis-understood but as above, I think a class-level (ie. something that implements either `RequestHandler` or `StreamingRequestHandler` - at any-point in its hierarchy) should just be the full-qualified class name; the method-name does not need to be appended to the handler.

Right?

Reply:
Correct. We remove the special method `handleRequest` if the class implements Lambda handler interfaces in `isValidHandler`.

See test case `testHandlerIsReturned_customMethodInRequestHandlerImplementation`